src/OSPSuite.Core/OSPSuite.Core.csproj

```diff
@@ -44,7 +44,6 @@
     <PackageReference Include="Microsoft.Extensions.Logging.Abstractions" Version="3.1.0" />
     <PackageReference Include="OSPSuite.Serializer" Version="3.0.0.1" />
     <PackageReference Include="OSPSuite.FuncParser" Version="4.0.0.68" GeneratePathProperty="true" />
-    <PackageReference Include="OSPSuite.SimModel" Version="4.0.0.68" GeneratePathProperty="true" />
```

Comment:
I somehow made a complete mess of updating SimModel and CVODES in this branch previously.